Source/RapyutaSimulationPlugins/Public/Core/RRMathUtils.h

```diff
@@ -0,0 +1,25 @@
+// Copyright 2020-2021 Rapyuta Robotics Co., Ltd.
+#pragma once
+
+// Native
+#include "math.h"
+
+// UE
+#include "Kismet/BlueprintFunctionLibrary.h"
+#include "Math/RandomStream.h"
+//#include "Math/UnrealMath.h"
+//#include "GenericPlatform/GenericPlatformMath.h"
+
+#include "RRMathUtils.generated.h"
+
+UCLASS()
+class RAPYUTASIMULATIONPLUGINS_API URRMathUtils : public UBlueprintFunctionLibrary
+{
+    GENERATED_BODY()
+
+public:
+    FORCEINLINE static FVector ConvertHandedness(const FVector& InLocation)

```

Comment:
This method may not necessary since we have ConversionUtils

Reply:
Well, it is actually needed and used currently to convert coordinate in Assimp to UE. ConversionUtils only do ROS<->UE conversion and implicitly include this handedness, but not all cases that we need to do 100 cm<->m conversion.
This function may not rightfully belong to MathUtils and rather be moved to ConversionUtils, but I prefer to keep the class since anyway upcoming methods will come soon.